tests/regression/77-lin2vareq/32-divbzero-in-overflow.c

```diff
@@ -0,0 +1,18 @@
+// SKIP PARAM: --set ana.activated[+] lin2vareq
+
+/**
+ * This test shows an instance where MaySignedOverflow raised
+ * an uncaught division by zero in the treatment of c's body
+ * 
+ * Fixed in #1419
+ */
+int a;
+char b;
+int c() { 0 == a && 1 / b; }
+void d(int (*func)()) { func(); }
+int main()
+{
+    d(c);
```

Comment:
Does the example still break before this fix if you inline the body of `c`?
If yes, it might make sense to do this to make the example easier.

Reply:
yes, it helps. I overestimated creduce's smartness.